crates/forge/tests/integration/setup_fork.rs

```diff
@@ -272,3 +273,33 @@ fn fork_get_block_info_fails() {
         "Unable to get block with tx hashes from fork"
     );
 }
+
+#[test]
+// found in: https://github.com/foundry-rs/starknet-foundry/issues/1175
+fn problematic_contract_fork() {
```

Comment:
`fn incompatible_abi` maybe?

Reply:
Also do we have a test against a fork with stored contract code in another sierra version? To test the `get_compiled_contract_class` from `ForkStateReader` (it was the first reason that caused us to switch to universal sierra compilation, right?)

Reply:
This one is in the 0.1.0